.flake8

```diff
@@ -1,2 +1,2 @@
 [flake8]
-exclude = .git,__pycache__,build,dist,versioneer.py,mongo/_version.py,docs/source/conf.py,suitcase/mongo_layout2/*
+exclude = .git,__pycache__,build,dist,versioneer.py,mongo/_version.py,docs/source/conf.py
```

Comment:
`mongo/_version.py` is outdated.

Reply:
I'm not sure what to change here

Reply:
I think `mongo/_version.py` is left over from an early iteration before suitcase-mongo had two subpackages. This should be `suitcase/mongo_normalized/_version.py, suitcase/mongo_embedded/_version.py` I think.

Reply:
updated